fed/api.py

```diff
@@ -116,48 +115,20 @@ def init(
                     "cert": "bob's server cert",
                     "key": "bob's server cert key",
                 }
-
         logging_level: optional; the logging level, could be `debug`, `info`,
             `warning`, `error`, `critical`, not case sensititive.
-        cross_silo_grpc_retry_policy: a dict descibes the retry policy for
-            cross silo rpc call. If None, the following default retry policy
-            will be used. More details please refer to
-            `retry-policy <https://github.com/grpc/proposal/blob/master/A6-client-retries.md#retry-policy>`_. # noqa
-
+        enable_waiting_for_other_parties_ready: ping other parties until they
+            are all ready if True.
+        global_cross_silo_comm_config: Global cross-silo communication related
+            config that are applied to all connections. Supported configs
+            can refer to CrossSiloCommConfig in config.py.
+        dest_party_comm_config: Communication config for the destination party
```

Comment:
`dest_party_comm_config` -> `single_party_comm_config` ?

Reply:
Has move this param into the `cluster` param, which previously contains dest party configs.